examples/gpio/gpio.c

```diff
@@ -0,0 +1,140 @@
+/* gpio.c
+ *
+ * Copyright (C) 2006-2021 wolfSSL Inc.
+ *
+ * This file is part of wolfTPM.
+ *
+ * wolfTPM is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * wolfTPM is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA
+ */
+
+/* This examples demonstrates the use of GPIO available on some TPM modules */
+
+#include <wolftpm/tpm2_wrap.h>
+
+#ifndef WOLFTPM2_NO_WRAPPER
```

Comment:
You'll get a build error with missing TPM2_GPIO_Config if these are not also defined `#if defined(WOLFTPM_ST33) || defined(WOLFTPM_AUTODETECT)`. Might consider making `TPM2_GPIO_Config` always available and add the ST33 specific stuff inside the function. When not compiled in use `NOT_COMPILED_IN` return code.